sql/updates/world/master/2018_10_04_00_world.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS `creature_template_model`;
+CREATE TABLE `creature_template_model`(
+    `entry` INT UNSIGNED NOT NULL,
+    `CreatureDisplayID` INT UNSIGNED NOT NULL,
+    `Probability` FLOAT(1) UNSIGNED NOT NULL,
```

Comment:
I'm still sure that this is not Probability more like ModelidXScale:

NPC: 131983

[0] CreatureDisplayID: 42178
[0] Probability: 1.25
[0] ProbabilityAlt: 1

[8] UNIT_FIELD_DISPLAYID: 42178/5.910397E-41
[8] UNIT_FIELD_DISPLAYSCALE: 1067450368/1.25

Reply:
I confirm, also UNIT_FIELD_DISPLAYSCALE and UNIT_FIELD_NATIVE_XDISPLAY_SCALE are always same
```
NPCID : 138812
[265] UNIT_FIELD_DISPLAYID: 78855/1.104994E-40
[265] UNIT_FIELD_DISPLAY_SCALE: 1065353216/1
[265] UNIT_FIELD_NATIVEDISPLAYID: 78855/1.104994E-40
[265] UNIT_FIELD_NATIVE_XDISPLAY_SCALE: 1065353216/1

NPCID : 131817
[3] UNIT_FIELD_DISPLAYID: 78855/1.104994E-40
[3] UNIT_FIELD_DISPLAY_SCALE: 1074790400/2.25
[3] UNIT_FIELD_NATIVEDISPLAYID: 78855/1.104994E-40
[3] UNIT_FIELD_NATIVE_XDISPLAY_SCALE: 1074790400/2.25
```